airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -92,32 +96,45 @@ def __init__(
         *,
         cluster_name: str,
         create_cluster_kwargs: dict | None = None,
-        wait_for_completion: bool = True,
+        wait_for_completion: bool = False,
```

Comment:
Does this require a deprecation warning since it's changing default behavior?

Reply:
Ooops. I've just incidentally change it by Ctrl+C + Ctrl + V 🤣 
I will revert it to previous value